### ISSUE TITLE
Postpone 2.475 weekly release from 9/3 to 9/4

### ISSUE THREAD

**Issue by basil** [jenkins-infra/helpdesk#4274]
### Service(s)

release.ci.jenkins.io

### Summary

https://github.com/jenkinsci/jenkins/pull/9672 is still waiting on approval from @daniel-beck, and he informed me it is unlikely he will be able to approve in time for a 9/3 weekly. Can we postpone the weekly to 9/4 to give Daniel more time? My concern with landing in 2.476 on 9/10 is that this will provide too little time for weekly testing before LTS selection. If we can ship in 2.475 that would be ideal. What do you think @timja @dduportal?

**Comment by dduportal** [comment]
I believe it should be possible yes. However we have to be careful in that case as the LTS `2.462.2` is also planned for the 4 Sept.

Ping @NotMyFault : if you feel like we can start the LTS release early wednesday morning (European time), then it's OK for us on infra.

**Comment by NotMyFault** [comment]
> Ping @NotMyFault : if you feel like we can start the LTS release early wednesday morning (European time), then it's OK for us on infra.

Surely, I'd start the build around 7am, if that works for you? GMT+2 ofc.

**Comment by basil** [comment]
Thanks! @dduportal @NotMyFault Do we need to disable the weekly release job for tomorrow morning and re-enable it for Wednesday, and if so can I rely on one of you to do that?

**Comment by timja** [comment]
Disabled in https://github.com/jenkins-infra/release/pull/591

**Comment by dduportal** [comment]
> Disabled in [jenkins-infra/release#591](https://github.com/jenkins-infra/release/pull/591)

Thanks @timja !

**Comment by dduportal** [comment]
> > Ping @NotMyFault : if you feel like we can start the LTS release early wednesday morning (European time), then it's OK for us on infra.
> 
> Surely, I'd start the build around 7am, if that works for you? GMT+2 ofc.

Looks good to me ! I'll be around while making breakfast if anything goes wrong. Thanks Alex

**Comment by dduportal** [comment]
For info: the LTS release 2.462.2 is finished (release + package): we can proceed to the weekly release

**Comment by dduportal** [comment]
https://github.com/jenkinsci/jenkins/pull/9672 is merged: let's roll!

**Comment by dduportal** [comment]
Both releases had been performed successfully today!

Weekly 2.475 includes the https://github.com/jenkinsci/jenkins/pull/9672 change.

Many thanks all!
